ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -84,23 +84,23 @@
       - conditional-step:
           condition-kind: shell
           condition-command: |
-            echo "${GIT_BRANCH}" | grep -v '\(reef\|squid\|centos9-only\|crimson-only\|jaeger\)'
+            echo "${{GIT_BRANCH}}" | grep -v '\(luminous\|mimic\|nautilus\|octopus\|pacific\|quincy\|reef\|squid\|centos9-only\|crimson-only\|jaeger\)'
```

Comment:
👍

Reply:
But maybe a comment about why?

Reply:
er...actually this was a mistake, and I think it should not be there